portal/src/graphql/adminapi/UserDetailCommandBar.tsx

```diff
@@ -43,12 +36,24 @@ const UserDetailCommandBar: React.FC<UserDetailCommandBarProps> = function UserD
         },
       },
       /* TODO: to be implemented
+      {
+        key: "loginAsUser",
+        text: renderToString("UserDetails.command-bar.login-as-user"),
+        iconProps: { iconName: "FollowUser" },
+      },
+      */
       {
         key: "disable",
         text: renderToString("disable"),
         iconProps: { iconName: "CircleStop" },
+        onRender: (props) => {
+          return (
+            <TodoButtonWrapper>
+              <CommandButton {...props} />
```

Comment:
Can we render them as disabled state for now?

Reply:
Updated

<img width="392" alt="Screenshot 2020-10-08 at 5 01 15 PM" src="https://user-images.githubusercontent.com/54486231/95437764-f9233080-0987-11eb-8d43-1132b3b8ef31.png">